src/lib/components/transaction/Transaction.svelte

```diff
@@ -82,6 +82,17 @@
     }
 
     $operationsVisibility = transactionGroups.map(() => false);
+
+    const minimumPopupHeight = 570;
+    const minimumPopupWidth = 360;
+
+    if (window.outerHeight < minimumPopupHeight) {
+        self.resizeTo(self.outerWidth, minimumPopupHeight);
+    }
+
+    if (window.outerWidth < minimumPopupWidth) {
+        self.resizeTo(minimumPopupWidth, self.outerHeight);
+    }
```

Comment:
This does not belong here. The Transaction component has no concept of a `window`. The Sign route would be more appropriate.